addons/refuel/CfgVehicles.hpp

```diff
@@ -512,6 +512,16 @@ class CfgVehicles {
         GVAR(fuelCargo) = REFUEL_INFINITE_FUEL;
     };
 
+    // Helper object for non-AllVehicles objects
+    class GVAR(helper): Helicopter_Base_F {
+        author = "Beswick.T";
+        scope = 1;
+        model = QPATHTOF(data\helper.p3d);
```

Comment:
Is this an actual model or just an empty dummy? One can use `model = "-";` instead for no model.